api/ephemeral-access/v1alpha1/accessrequest_types.go

```diff
@@ -46,6 +46,9 @@ const (
 
 	// InvalidStatus is the used to identify invalid access requests
 	InvalidStatus Status = "invalid"
+
+	// TimeoutStatus is the stage that defines the access request as timed out
+	TimeoutStatus Status = "timeout"
```

Comment:
It feels like it should be "timed-out", "expired-request" or perhaps transition to the "expired" status with a reason that the "request has timed out waiting for access to be granted".

Reply:
This is subjective and clear enough to me. The code has plenty of documentation and is self-explanatory to avoid confusion.